### DISCUSSION TITLE
FutureWarning: The behavior of 'to_datetime' with 'unit' when parsing strings is deprecated

### DISCUSSION THREAD

**Discussion by MirekF** [whittlem/pycryptobot#784] [General]
Hello,

After installing version 8.02, I'm seeing error messages when starting the application. Can you fix it?

/models/exchange/binance/api.py:1037: FutureWarning: The behavior of 'to_datetime' with 'unit' when parsing strings is deprecated. In a future version, strings will be parsed as datetime strings, matching the behavior without a 'unit'. To retain the old behavior, explicitly cast ints or floats to numeric type before calling to_datetime.
  pd.to_datetime(df["open_time"], unit="s"),

/models/exchange/binance/api.py:1046: FutureWarning: The behavior of 'to_datetime' with 'unit' when parsing strings is deprecated. In a future version, strings will be parsed as datetime strings, matching the behavior without a 'unit'. To retain the old behavior, explicitly cast ints or floats to numeric type before calling to_datetime.
  tsidx = pd.DatetimeIndex(pd.to_datetime(df["open_time"], unit="s"), dtype="datetime64[ns]")

**Comment by qq020608** [comment]
back to   pandas 1.5.1

**Comment by whittlem** [comment]
Thanks, yes... Pandas 2.* is starting to show Warnings. The version you want to be using is 1.5.1.
